lib/plugins/aws/invokeLocal/index.js

```diff
@@ -157,7 +158,8 @@ class AwsInvokeLocal {
     });
 
     if (process.env.VIRTUAL_ENV) {
-      process.env.PATH = `${process.env.VIRTUAL_ENV}/bin:${process.env.PATH}`;
+      const runtimeDirectory = os.platform() === 'win32' ? 'Scripts' : 'bin';
+      process.env.PATH = `${process.env.VIRTUAL_ENV}/${runtimeDirectory}:${process.env.PATH}`;
```

Comment:
Should we use `path.join()` here?

Not sure how `PATH` is handled on Windows machines, but it could be a problem because of `/` vs. `\`.

Reply:
🤔 You might be right. Path is always safer than concatenation/template literals. Fix incoming.

Reply:
Nice! Thanks @RafalWilinski 👍